app/src/main/java/com/dropbox/android/sample/Graph.kt

```diff
@@ -48,10 +67,40 @@ object Graph {
         return SourcePersisterFactory.create(cacheDir)
     }
 
+    fun provideConfigStore(context: Context): Store<Unit, RedditConfig> {
+        val fileSystem = FileSystemFactory.create(context.cacheDir)
+        val fileSystemPersister = FileSystemPersister.create(fileSystem, object : PathResolver<Unit> {
+            override fun resolve(key: Unit) = "config.json"
+        })
+        val adapter = moshi.adapter<RedditConfig>(RedditConfig::class.java)
+        return StoreBuilder
+                .fromNonFlow<Unit, RedditConfig> {
+                    delay(500)
+                    RedditConfig(10)
+                }
+                .nonFlowingPersister(
```

Comment:
@pavlospt this is a good starting point for what integration with `FileSystem` looks like. Feels like we can make this nicer/in a wrapper

Reply:
@digitalbuddha indeed. Will try to study it a bit more and give it a try with a wrapper based on this. Sorry for the delay, it's just that I have not yet been that familiar with the new Store 😛